common/app-starters-security-common/src/test/java/org/springframework/cloud/stream/app/security/common/SecurityCommonTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.app.security.common;
+
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.support.BasicAuthenticationInterceptor;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Christian Tzolov
+ * @author Artem Bilan
+ */
+@RunWith(Enclosed.class)
+public class SecurityCommonTest {
+
+
+	@TestPropertySource(properties = {
+			"spring.cloud.security.enabled=true",
+			"management.endpoints.web.exposure.include=health,info,env",
+			"info.name=MY TEST APP"})
+	public static class SecurityEnabledManagementSecurityEnabledTests extends AbstractSecurityCommonTests {
+
+		@Test
+		@SuppressWarnings("rawtypes")
+		public void testHealthEndpoint() {
+			ResponseEntity<Map> response = this.restTemplate.getForEntity("/actuator/health", Map.class);
+			assertEquals(HttpStatus.OK, response.getStatusCode());
+			assertTrue(response.hasBody());
+			Map health = response.getBody();
+			assertEquals("UP", health.get("status"));
+		}
+
+		@Test
+		@SuppressWarnings("rawtypes")
+		public void testInfoEndpoint() {
+			ResponseEntity<Map> response = this.restTemplate.getForEntity("/actuator/info", Map.class);
+			assertEquals(HttpStatus.OK, response.getStatusCode());
+			assertTrue(response.hasBody());
+			Map info = response.getBody();
+			assertEquals("MY TEST APP", info.get("name"));
+		}
+
+		// The ManagementWebSecurityAutoConfiguration exposes only Info and Health endpoint not Env!
```

Comment:
So, looks like we need also to come up with something to make `/bindings` visible as well...
Since an idea to have it unsecured by default, too.

Am I missing anything ?

Can be as a separate issue though...

Reply:
It seems like the `bindings` has never been used by SCDF.

Reply:
It looks like `/bindings` has never been used in SCDF

Reply:
OK. Let's raise such an issue against @sabbyanandan !

Although we still have it in the properties in the Maven plugin: https://github.com/spring-cloud/spring-cloud-app-starters-maven-plugins/blob/master/spring-cloud-stream-app-maven-plugin/src/main/java/org/springframework/cloud/stream/app/plugin/SpringCloudStreamAppMojo.java#L335

So, sounds like this one has to be unsecured as well.

As I said: we can revise it in the separate issue.

Reply:
noticed this generated properties and started a discussion in the scdf-core-team channel but no one managed to justify the reason of exposing this endpoint. There is not reference of being used in neither of the SCDF projects i've checked